src/components/Clock.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+function getRandomName(): string {
+  const value = Date.now().toString().slice(-4);
+
+  return `Clock-${value}`;
+}
+
+function stringDate(info: Date) {
+  return info.toUTCString().slice(-12, -4);
+}
+
+type State = {
+  today: Date;
+  clockName: string;
+};
+
+type Props = {
+  status: boolean;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    today: new Date(),
+    clockName: 'Clock-0',
+  };
+
+  componentDidMount(): void {
+    window.setInterval(this.timerId, 3300);
+    window.setInterval(this.updateDate, 1000);
+  }
+
+  componentDidUpdate(
+    _prevProps: Readonly<{}>,
+    prevState: Readonly<{ clockName: string }>,
+  ): void {
+    if (prevState.clockName !== this.state.clockName) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${prevState.clockName} to ${this.state.clockName} `);
+    }
+  }
+
+  timerId = () => {
+    this.setState({ clockName: getRandomName() });
+  };
+
+  updateDate = () => {
+    this.setState({ today: new Date() });
+    if (this.props.status) {
+      // eslint-disable-next-line no-console
+      console.info(stringDate(this.state.today));
+    }
+  };
+
+  render() {
+    return (
+      <>
+        {this.props.status && (
+          <div className="Clock">
+            <strong className="Clock__name">{this.state.clockName}</strong>
```

Comment:
Don't use upper case for the first letter, better practice is to start the name from the lower letter. Check other similar cases.

Reply:
This names was created in empty project. If I change this names, use lower case, all test fall.